packages/components/src/components/date-picker/_flatpickr.scss

```
@@ -142,6 +142,7 @@
   width: rem(40px);
   padding: 0;
   fill: $icon-01;
+  transform: scale(1, 1) /* rtl: scale(-1,1) */;
```

Due to a limitation with the way sass compiles RTL directives, we need to change the format to something like this so it is correctly applied 

```suggestion
  transform: scale(1, 1) #{'/*rtl: scale(-1,1)*/'};
```

See usage in Checkbox: https://github.com/carbon-design-system/carbon/blob/main/packages/components/src/components/checkbox/_checkbox.scss#L105

---

Hi @tw15egan ,
Thanks for your review, I had updated the code as suggested.